apps/coscrad-frontend/src/components/resources/digital-text/page-publication-form.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button } from '@mui/material';
+import { useState } from 'react';
+
+interface PublicationFormProps {
+    onSubmitForPublication: () => void;
+}
+
+/**
+ * TODO Move this to a higher level directory
```

Comment:
We should do this when we add an interactive command flow for a second resource type.